src/psyclone/profiler.py

```diff
@@ -42,10 +42,10 @@
 from __future__ import absolute_import, print_function
 from psyclone.f2pygen import CallGen, TypeDeclGen, UseGen
 from psyclone.psyGen import GenerationError, Kern, NameSpace, \
-     NameSpaceFactory, Node, BuiltIn, InternalError
+     NameSpaceFactory, Node, InternalError
 
 
-class Profiler():
+class Profiler(object):
```

Comment:
Joerg and I are moving to removing things that cause a warning under Python 2 so we wouldn't bother with `object` here (unless its absence breaks something).

Reply:
OK.